addons/mrp/report/mrp_production_templates.xml

```diff
@@ -117,7 +117,7 @@
                         <t t-set="move_lines" t-value="move.move_line_ids.filtered(lambda x: x.state != 'done' and x.product_qty)"/>
                     </t>
                     <t t-foreach="move_lines" t-as="move_line">
-                        <t t-if="move_line.product_uom_id.category_id.measure_type == 'unit'">
+                        <t t-if="move_line.product_uom_id.category_id == production.env.ref('uom.product_uom_categ_unit')">
```

Comment:
Sure that 'self' or just ref/env won't work instead of production and picking.env...?  (just to avoid confusion)  Maybe it is  not that common to use xml_ids in qweb.